pom.xml

```diff
@@ -29,21 +29,21 @@
     <module>waggle-dance-boot</module>
     <module>waggle-dance-integration-tests</module>
     <module>waggle-dance</module>
-    <module>waggle-dance-rpm</module>
+<!--    <module>waggle-dance-rpm</module>-->
```

Comment:
I'm guessing you're having some problem building this? What's not working for you?